.travis.yml

```diff
@@ -2,7 +2,6 @@ language: vim
 
 before_script: |
   git clone https://github.com/junegunn/vader.vim.git
-  git clone https://github.com/soywod/kronos.vim.git
```

Comment:
Hum weird, I dunno how it came .. xD thank you :+1:

Reply:
No problem :)